polkadot-parachain/src/legacy_chain_spec/mod.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+#![allow(dead_code)]
+
```

Comment:
add some comment about the origin of this `mod` and why it is here.

Reply:
https://github.com/paritytech/cumulus/pull/2936/commits/d2a7497d5090d47e952df472dc77dfa371b1d081